ll/stack.java

```diff
@@ -0,0 +1,38 @@
+import java.io.*;
+class stack{
+  public static void main(String args[])
+  {
+    int value;
+System.out.println("implementation of Stack:-");
+LinkedList list=new LinkedList();
+BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+try
+{
+while(true)
+{
+System.out.println("1.push 2.pop 3.print 4.exit ");
+int n=Integer.parseInt(br.readLine());
+switch(n)
+{
+case 1:System.out.println("enter value");
+       value=Integer.parseInt(br.readLine());       
+        list.insertAtTail(value);       

```

Comment:
@Abhijeet-prakash This works, but if u insert at tail,while deleting at tail,u will have to traverse the whole list, TC O(n). But if u insert at head, u can delete head,which takes O(1) time.